lib/pages/approval/view_approvals.dart

```diff
@@ -100,7 +100,7 @@ class _ViewApprovalsState extends State<ViewApprovals> {
                   //add title of the video (bolded)
                   arts.add(Text(data['name'],
                       style: const TextStyle(fontWeight: FontWeight.bold)));
-                  artSnapshot.data!.docs.forEach((element) {
```

Comment:
I am curious as to why this is being changed?

Reply:
because flutter doesn't like it. I don't know why